### ISSUE TITLE
Indication in offense count report of autocorrectable offenses

### ISSUE THREAD

**Issue by fragkakis** [rubocop/rubocop#12273]
## Is your feature request related to a problem? Please describe.

When I run rubocop and generate the offense count report on my codebase it is not clear to me which cops can be autocorrected. Furthermore, when I mass-autocorrect everything, it is not clear in the subsequent diff which offenses are corrected with each change.

## Describe the solution you'd like

I would like a clear indication in the offense count report to see which violations can be autocorrected. Ideally a different indication for the ones that can be autocorrected safely vs the ones that can be autocorrected not safely. This way I can try and autocorrect the offenses 1 by 1, and have a clear picture of how each correction changes the codebase.

## Describe alternatives you've considered

-

## Additional context

For example, this report:

```
...
2     Layout/FirstArrayElementIndentation
2     Layout/MultilineMethodCallIndentation
2     Lint/DuplicateCaseCondition
2     Lint/DuplicateRescueException
2     Lint/UnreachableCode
2     Performance/BindCall
2     Performance/CompareWithBlock
...
--
3954  Total in 1566 files
```

could become:

```
...
2     Layout/FirstArrayElementIndentation (safely autocorrectable)
2     Layout/MultilineMethodCallIndentation
2     Lint/DuplicateCaseCondition
2     Lint/DuplicateRescueException  (unsafely autocorrectable)
2     Lint/UnreachableCode  (safely autocorrectable)
2     Performance/BindCall  (unsafely autocorrectable)
2     Performance/CompareWithBlock
...
--
3954  Total in 1566 files
```

or, more concisely:

```
...
2     Layout/FirstArrayElementIndentation (a)
2     Layout/MultilineMethodCallIndentation
2     Lint/DuplicateCaseCondition
2     Lint/DuplicateRescueException  (A)
2     Lint/UnreachableCode  (a)
2     Performance/BindCall  (A)
2     Performance/CompareWithBlock
...
--
3954  Total in 1566 files
```

**Comment by fragkakis** [comment]
Thank you for taking this into account! It will make rubocop much easier to mass-apply.
